webapp/src/main/java/com/box/l10n/mojito/utils/NamingTransformation.java

```diff
@@ -0,0 +1,66 @@
+package com.box.l10n.mojito.utils;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import io.micrometer.core.instrument.util.StringUtils;
+
+public enum NamingTransformation {
```

Comment:
what does NamingTransformation do exactly? Why do we need this? Can you add some tests?